readthedocs/rtd_tests/fixtures/spec/v2/schema.json

```diff
@@ -126,6 +132,28 @@
                 "type": "string"
               }
             },
+            "build": {
+              "description": "Override the default build process.",
+              "type": "object",
+              "additionalProperties": false,
+              "required": ["html"],
+              "properties": {
```

Comment:
Do we not want to support epub and htmlzip here?

Reply:
Sorry, forgot to update the PR description with my local notes... Updated. But basically I'm fine adding support for all formats.

Reply:
I added support for all formats.